### DISCUSSION TITLE
Connection to SDR Dongle

### DISCUSSION THREAD

**Discussion by KGx55** [AlexandreRouma/SDRPlusPlus#725] [General]
I am looking at the code and trying to get a sense of how it works.  While I have at many files, I haven't yet seen the one that connects to the dongle. Where can I find that?

**Comment by the-gobleo-tinkerer** [comment]
They are in the [source_modules](https://github.com/AlexandreRouma/SDRPlusPlus/tree/master/source_modules) folder.
